nservicebus/hosting/ibus-instance.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
I agree with Simon. Add the following at the top: 

In typical NServiceBus projects the IBus is obtained by the automatic configured injection as explained below. 
However in some applications one may wish to use a different approach :
* Typically for web applications, the project can be configured with an IBus for one-way messaging <link needed to explain this> 
** Ad hoc IBus creation - for a quick sample < code: using (...){}  >, 
** Manual configuration of the IBus injection in a typical web app 
*In SignalR web applications configure the IBus injection manually    

***Automatically Configured Injection***
The following code is needed for acquiring an injected IBus in your handler classes. 
Note: The configuration code is automatically added, if you use the Nuget for the NServiceBus Host package. 

< here goes all the current injecting-bus documentation >

** Manually setting IBus injection **
< here goes you new doc >